mentors/python/z.json

```
@@ -1,4 +1,12 @@
 [
+  {
+    "github_username": "zhengfu23",
+    "name": "James Fu",
+    "link_text": null,
+    "link_url": null,
+    "avatar_url": null,
+    "bio": "Python is one of my favorite languages. I fell in love with it seeing the beautiful absence of brackets and semi-colon. I also love it for list comprehension and its extensive use in Machine Learning and Raspberry Pi. I think everyone should learn Python!"
+  }
```

I think you're missing a comma
```suggestion
  },
```

---

Haha thanks!